sdk/python/README.md

```diff
@@ -46,14 +46,36 @@ Class | Method | Description
 [TFJobClient](docs/TFJobClient.md) | [is_job_succeeded](docs/TFJobClient.md#is_job_succeeded) | Check if the TFJob status is Succeeded |
 [TFJobClient](docs/TFJobClient.md) | [get_pod_names](docs/TFJobClient.md#get_pod_names) | Get pod names of TFJob |
 [TFJobClient](docs/TFJobClient.md) | [get_logs](docs/TFJobClient.md#get_logs) | Get training logs of the TFJob |
+[PyTorchJobClient](docs/PyTorchJobClient.md) | [create](docs/PyTorchJobClient.md#create) | Create PyTorchJob|
```

Comment:
I believe the Client docs were deleted by generator. Should we modify our script to not delete `PyTorchJobClient.md` and `docs/TFJobClient.md` during SDK generator run ?

Reply:
Thanks for catching that, added client docs back